### DISCUSSION TITLE
好用，就是有没有全局白名单的功能，有时候会在其他群误触有点烦

### DISCUSSION THREAD

**Discussion by limelisest** [Tsuk1ko/CQ-picfinder-robot#167] [我有想法]
目前只看到setu功能是有白名单的，看了一圈wiki和配置文件也没找到白名单)

**Comment by Tsuk1ko** [comment]
会考虑，得设计下逻辑

**Reply by limelisest** [reply]
还有一个问题是，以图搜图的时候会先发送一次搜索失败，然后再发送结果。

**Reply by Tsuk1ko** [reply]
看日志，是不是到 saucenao 网络有问题

**Comment by Tsuk1ko** [comment]
突然发现其实 [go-cqhttp 的事件过滤器](https://docs.go-cqhttp.org/guide/eventfilter.html#%E5%8F%AA%E4%B8%8A%E6%8A%A5%E7%A7%81%E8%81%8A%E6%88%96%E7%89%B9%E5%AE%9A%E7%BE%A4%E7%BB%84%E7%9A%84%E9%9D%9E%E5%8C%BF%E5%90%8D%E6%B6%88%E6%81%AF)就可以实现你的需求，那我就摸了（
